### ISSUE TITLE
Demo doesn't work

### ISSUE THREAD

**Issue by gustavjf** [tomchentw/react-toastr#31]
Just wanted to mention I tried the demo but the `Hello` button doesn't seem to do anything.

Console output:
```
http://tomchentw.github.io/react-toastr/assets/7b03ecb9a7b44d9a67fb/client.css Failed to load resource: the server responded with a status of 404 (Not Found)
client.js:17 Uncaught Error: Cannot find module "-!./../node_modules/css-loader/index.js?root=..!toastr/toastr.css"
http://tomchentw.github.io/favicon.ico Failed to load resource: the server responded with a status of 404 (Not Found)
http://tomchentw.github.io/react-toastr/assets/7b03ecb9a7b44d9a67fb/client.css Failed to load resource: the server responded with a status of 404 (Not Found)
```

**Comment by gustavjf** [comment]
Looks like it works now, thanks for the fix!

**Comment by tomchentw** [comment]
Sorry about that. Looks like it had been resolved via the release of [v2.0.0](https://github.com/tomchentw/react-toastr/releases/tag/v2.0.0)
